src/util/c_types.h

```diff
@@ -11,6 +11,7 @@ Author: Daniel Kroening, kroening@kroening.com
 #define CPROVER_UTIL_C_TYPES_H
 
 #include "std_types.h"
+#include "mathematical_types.h"
```

Comment:
I guess that's really just for `complex_typet`? Maybe keep that in `std_types.h`?

Reply:
It's a mathematical type so if we're moving some of them into a separate file, it only makes sense if we really move all of them. Is there any particular reason why this stands out for you?

Reply:
I thought it was the one that's actually used by C/C++ front-ends, but maybe it's not even the only one.

Reply:
You are right, there is `std::complex` type, moving back to `std_types`.